src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { getUser } from '../../api';
+
+import { Todo } from '../../types/Todo';
+import { TodoModal } from '../TodoModal';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const showModal = () => setModalIsOpen(!modalIsOpen);
+  const {
+    title,
+    id,
+    completed,
+    userId,
+  } = todo;
+
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(userId)
+      .then(userData => {
+        setUser(userData);
+      });
+  }, [userId]);
+
+  return (
+    <>
+      <tr
+        data-cy="todo"
+        className={classNames({ 'has-background-info-light': modalIsOpen })}
+        key={id}
```

Comment:
The `key` prop is not necessary and can be removed because `key` is only needed when components are rendered inside a loop.